feat: Add Installation Guide for @sentry/capacitor (#3730)
getsentry/sentry-docs: src/includes/getting-started-config/javascript.capacitor.mdx

> You should `init` the SDK as soon as possible during your app's creation stage and forward the `init` method from the sibling Sentry SDK for the framework you use.

Reviewer: Some of this becomes redundant when you preview, since the common content includes basic prose. Edited to minimize the redundancy:
```suggestion
Then forward the `init` method from the sibling Sentry SDK for the framework you use, such as Angular in this example:
```

```typescript {tabTitle: Angular} {filename: app.module.ts}
import * as Sentry from "@sentry/capacitor";
import * as SentryAngular from "@sentry/angular";

Sentry.init(
  {
    dsn: "___PUBLIC_DSN___",

    // To set your release and dist versions

>     release: "my-project-name@" + process.env.npm_package_version,

Author: @bruno-garcia This was a carry-over from the previous version of this page. I think it would make more sense to just put it as `<release>` given we refer to it that way in the source map upload command below?

>     release: "my-project-name@" + process.env.npm_package_version,

Author: or maybe `"my-project-name@" + <release_name>`

>     release: "my-project-name@" + process.env.npm_package_version,

Reviewer 2: Sounds good!

    dist: "1"
  },
  // Forward the init method from @sentry/angular
  SentryAngular.init
);

@NgModule({
  providers: [
    {
      provide: ErrorHandler,
      // Attach the Sentry ErrorHandler
      useValue: SentryAngular.createErrorHandler(),
    },
  ],
})
```

```javascript {tabTitle: Other Frameworks}
import * as Sentry from "@sentry/capacitor";

Sentry.init({
  dsn: "___PUBLIC_DSN___",

  // To set your release version
  release: "my-project-name@" + process.env.npm_package_version,
});
```

> You can also use all the features that comes with each Sentry SDK for the framework you use such as Angular. You can read more about them over at [Sentry For Angular](/platforms/javascript/guides/angular/).

Reviewer: ```suggestion
You can also use the features available with the Sentry SDK for the framework you use, such as [Angular](/platforms/javascript/guides/angular/).
```

### Source Maps

You will need to upload source maps to make sense of the events you receive in Sentry.

> For example, if using Capacitor with Ionic-Angular, you would upload your `www` folder on **every build**. The values for `<release_name>` and `<dist>` will need to match the values you pass into `Sentry.init` for events to be deminified correctly.

Reviewer: ```suggestion
For example, if you are using Capacitor with Ionic-Angular, upload your `www` folder on **every build**. The values for `<release_name>` and `<dist>` must match the values passed into `Sentry.init` for events to be deminified correctly.
```

```bash {tabTitle: Ionic}
sentry-cli releases files <release_name> upload-sourcemaps ./www --dist <dist>
```

> You can read more about uploading sourcemaps over on the [Source Maps page](/platforms/javascript/guides/capacitor/sourcemaps/).

Reviewer: ```suggestion
Learn more about uploading [source maps](/platforms/javascript/guides/capacitor/sourcemaps/).
```
